lib/schmobile/middleware.rb

```diff
@@ -31,7 +31,7 @@ def redirect?(request)
         redirecting = false
       end
 
-      redirecting ? redirect_location(request) : nil
```

Comment:
Hold on. I think this `nil` was intentional, and necessary. I'll need to investigate.